src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClient.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.CaseClientServiceException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.ServiceResponseParser;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.request.CaseUpdate;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.request.ExistingCaseDetails;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.SuccessfulUpdateResponse;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ExceptionRecord;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.UNPROCESSABLE_ENTITY;
+
+@Component
+public class CaseUpdateClient {
+
+    private static final Logger log = LoggerFactory.getLogger(CaseUpdateClient.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The constant name 'log' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4535703)